modules/users/client/controllers/admin/add-user.client.controller.js

```diff
@@ -3,18 +3,17 @@
 
   angular
     .module('users')
-    .controller('AuthenticationController', AuthenticationController);
+    .controller('AddUserController', AuthenticationController);
 
-  AuthenticationController.$inject = ['$scope', '$state', 'UsersService', '$location', '$window', 'Authentication', 'PasswordValidator', 'Notification'];
+  AddUserController.$inject = ['$scope', '$state', 'UsersService', '$location', '$window', 'Authentication', 'PasswordValidator', 'Notification'];
 
-  function AuthenticationController($scope, $state, UsersService, $location, $window, Authentication, PasswordValidator, Notification) {
+  function AddUserController($scope, $state, UsersService, $location, $window, Authentication, PasswordValidator, Notification) {
     var vm = this;
 
     vm.authentication = Authentication;
```

Comment:
we don't need all of this password validation and other stuff that the auth controller has.